### DISCUSSION TITLE
activiti7/1

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#780] [Announcements]
# activiti7/1

1.1 概念 工作流(Workflow)，就是通过计算机对业务流程自动化执行管理。它主要解决的是“使在多个参与者之间按照某种预定义的规则自动进行传递文档、信息或任务的过程，从而实现某个预期的业务目标，或者促使此目标的实现”。 1.2 工作流系统 一个软件系统中具有工作流的功能，我们把它称为工作流系统，一个系统中工作流的功能是什么？就是对系统的业务流程进...

https://doc.chenbufan.cn/activiti7/1.html

**Comment by blur-9527** [comment]
1

**Comment by blur-9527** [comment]
你们好啊！
